python/cudf/cudf/core/frame.py

```diff
@@ -1044,13 +1040,59 @@ def searchsorted(
         1-D cupy array of insertion points
         """
         # Call libcudf++ search_sorted primitive
-        outcol = libcudfxx.search.search_sorted(
+        outcol = libcudf.search.search_sorted(
             self, values, side, ascending=ascending, na_position=na_position
         )
 
         # Retrun result as cupy array
         return cupy.asarray(outcol.data_array_view)
 
+    def get_sorted_inds(self, ascending=True, na_position="last"):
```

Comment:
This shouldn't be exposed to classes that inherit from Frame.
```suggestion
    def _get_sorted_inds(self, ascending=True, na_position="last"):
```